frontend/src/lang/es.ts

```diff
@@ -412,10 +412,19 @@ export const es: LanguageConfig = {
         positive: {
           p0: "Por lo general, el virus respiratorio sincitial (VRS o RSV, por sus siglas en inglés) causa síntomas leves similares a los de un resfriado. La mayoría de las personas pueden recuperarse en casa, pero el VRS puede causar enfermedad grave y hospitalización en bebés y personas mayores. Usted puede ayudar a prevenir la propagación del VRS si se queda en casa cuando esté enfermo, evita el contacto cercano con otras personas y se lava las manos con frecuencia.",
           p1: "Puede tomar medidas para aliviar los síntomas, como beber suficientes líquidos y tomar medicamentos de venta sin receta para controlar el dolor y la fiebre. Si su hijo/a tiene el VRS, hable con su proveedor de atención médica antes de darle medicamentos para el resfriado sin receta.",
-          p2: "Comuníquese con un profesional de atención médica si sus síntomas empeoran, tiene problemas para respirar o está deshidratado. <0>Obtenga más información sobre los síntomas y cuidados de la infección por el VRS en el sitio web de los CDC</0>",
+          p2: "Comuníquese con un profesional de atención médica si sus síntomas empeoran, tiene problemas para respirar o está deshidratado. <0>Obtenga más información sobre los síntomas y cuidados de la infección por el VRS en el sitio web de los CDC</0> (espanol.cdc.gov/rsv/about/symptoms.html)",
           rsvSymptomsLink: "https://espanol.cdc.gov/rsv/about/symptoms.html",
         },
       },
+      hivNotes: {
+        h1: "Para el VIH:",
```

Comment:
@jayna-SkylightDigital I didn't see a translation for this so I used google translate 😅 -- please let me know if this needs to be updated to make sense 🙏